### ISSUE TITLE
Cannot find device "fluxwl0v"

### ISSUE THREAD

**Issue by yuginiwa** [FluxionNetwork/fluxion#614]
|    Q        |   A
|------------ | -----
| Version     | 5.6

#### Support Question

[<a href="https://ibb.co/g6ZJxz"><img src="https://preview.ibb.co/nuodxz/Screenshot_from_2018_09_01_11_21_56.png" alt="Screenshot_from_2018_09_01_11_21_56" border="0"></a>](url)

Hi. Can you help me about in this error. Since I update to 5.5 and 5.6 fluxwlOv cannot find. I'm using Fluxion 3 before and there no problem like that. Thank you.

**Comment by Tanuj69** [comment]
Facing the same issue

**Comment by Tanuj69** [comment]
React to my comment to ping me if solution is found

**Comment by deltaxflux** [comment]
Yeah I will take a look at this now sorry for the delay.

**Comment by deltaxflux** [comment]
Maybe deleting attacks/Captive Portal/attack.conf could help.Try it.

**Comment by deltaxflux** [comment]
Fixed?

**Comment by Tanuj69** [comment]
Yeah it is fixed

**Comment by adityacharate** [comment]
I have the same problem. I deleted the attack.conf file but the same issue persists. My wifi card is TP-link TLWN 722-n V1 and is capable of monitor mode.
![IMG_20200403_083725](https://user-images.githubusercontent.com/63085087/78320640-8c1d8100-7587-11ea-9e35-3be5b8dadf13.jpg)

**Comment by BoomerByte** [comment]
I also deleted it and still the problem

**Comment by TimeOfJudgment** [comment]
Deleting attack.conf fixed the issue

**Comment by 666-Christ-666** [comment]
> I have the same problem. I deleted the attack.conf file but the same issue persists. My wifi card is TP-link TLWN 722-n V1 and is capable of monitor mode. ![IMG_20200403_083725](https://user-images.githubusercontent.com/63085087/78320640-8c1d8100-7587-11ea-9e35-3be5b8dadf13.jpg)

Hi did you solve the issue ?
I have the exact same card and I am experiencing the same problem 🤢

**Comment by cjb900** [comment]
@666-Christ-666 instead of replying to a post from 2+ years ago it would help if you create a new issue with your exact set up and the exact issue you are having. Also if you are also using Zorin OS its possible something there may be the issue since that is not a dedicated pentesting Linux distro like Kali or Parrot OS is.

**Comment by 666-Christ-666** [comment]
> @666-Christ-666 instead of replying to a post from 2+ years ago it would help if you create a new issue with your exact set up and the exact issue you are having. Also if you are also using Zorin OS its possible something there may be the issue since that is not a dedicated pentesting Linux distro like Kali or Parrot OS is.

I am using the last version of Parrot OS and I have the exact same wifi card it works fine when I use my card (Wn722n) as a jamming device and my laptop as captive portal but when I try to use my wifi card for both it gives the error above maybe I should switch to Kali

**Comment by cjb900** [comment]
well as long as that card actually has the AR9271 chipset may others use that card with little to no issues. Its possible if the interface name of your wifi adapter is not the normal wlan0, wlan1 etc... that may be part of your issue. Also you can try rebooting your computer, unplug and replug in your wifi adapter and then before starting fluxion run ``sudo airmon-ng check kill`` and see if that helps.

**Comment by 666-Christ-666** [comment]
> well as long as that card actually has the AR9271 chipset may others use that card with little to no issues. Its possible if the interface name of your wifi adapter is not the normal wlan0, wlan1 etc... that may be part of your issue. Also you can try rebooting your computer, unplug and replug in your wifi adapter and then before starting fluxion run `sudo airmon-ng check kill` and see if that helps.

Yeah they are not like Wlan0,Wlan1 etc. how can I fix that ? is this a driver issue ? 
Check out how do they look 
![IMG_20220830_045851_831](https://user-images.githubusercontent.com/112444067/187331282-664640ca-0aba-449b-acc6-3274f4eb443c.jpg)

**Comment by cjb900** [comment]
well first what you want to do is get them back to not being any of the fluxion interface naming by rebooting your whole computer and unplugging and replugging in your wifi adapter. then run this command in a terminal ``sudo ln -s /dev/null /etc/udev/rules.d/80-net-setup-link.rules`` then reboot again. this should get the interface names to be wlan0, wlan1 etc... then run ``sudo airmon-ng check kill`` before starting fluxion and see if that helps with your issues.

**Comment by 666-Christ-666** [comment]
I just installed the latest kali linux and it works fine now expect the mdk4 option.

**Comment by TimeOfJudgment** [comment]
the latest kali linux everything is working great for the first time
